automated/linux/kunit/parse-output.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+import sys
+import re
+
+
+def slugify(line):
+    non_ascii_pattern = r"[^A-Za-z0-9]+"
```

Comment:
I think you can add _- here as well. LAVA is OK with underscores and hyphens. It doesn't like white spaces.

Reply:
done, thanks you for your input.